pkg/mod/github.go

```diff
@@ -0,0 +1,140 @@
+package mod
+
+import (
+	"context"
+	"log"
+	"os"
+	"os/user"
+	"path"
+	"path/filepath"
+	"strings"
+
+	"github.com/google/go-github/v32/github"
+	"github.com/pkg/errors"
+)
+
+var client *github.Client
```

Comment:
Does `client` have to be a global variable or could it be better encapsulated inside `githubMgr`